app/views/my/_batch_edits_actions.html.erb

```diff
@@ -1,2 +1,2 @@
-<li data-behavior="batch-edit-select-none" data-state="on"><a href="#"><i class="glyphicon glyphicon-ok"></i> Select None</a></li>
-<li data-behavior="batch-edit-select-page" data-state="off"><a href="#"><i class=""></i> Select Current Page</a></li>
+<li data-behavior="batch-edit-select-page" data-state="off"><a href="#">All</a></li>
```

Comment:
I'm not sure we ought to say "All" as we are only going to select things on this page, not the whole of the search results.  Saying "Select Current Page" keeps this clear.